config/plugin.json

```diff
@@ -0,0 +1,8 @@
+{
+	"features": {
+		"activity-panels": false,
+		"analytics": true,
+		"analytics/categories": false,
```

Comment:
No discussion around this. I think the plan is still to have a categories report -- I just wanted to provide an example here for a "sub-feature", since the category report is still using Swagger. I can remove this flag before merge.

Reply:
Removed the flag here since we won't end up needing it: https://github.com/woocommerce/wc-admin/pull/1500/commits/81607dc415c90a146867a373fdf157b892ff2631.